api/application.py

```diff
@@ -285,6 +289,52 @@ def get_interaction(self, interaction_id: str) -> InteractionInfoResponse:
             )
         )
 
+    @router.post("/interactions/{interaction_id}/scores")
+    def score_interaction(
+        self,
+        request: Request,
+        interaction_id: str,
+        score: InteractionScore,
+    ) -> ItemCreateResponse:
+        """
+        Give an interaction a feedback to measure if the result is good.
+
+        Args:
+            interaction_id (str): The ID of the interaction to score.
+            score (InteractionScore): The score detail
+
+        Returns:
+            ItemCreateResponse: An object indicating the success or failure of the score operation.
+        """
+
+        # get langfuse configuration
+        interaction = self.database.get_interaction(interaction_id)
+        if not interaction:
+            raise InteractionNotFound(interaction_id)
+        app = self.database.get_application(interaction.app_id)
+        if not app:
+            raise ApplicationNotFound(app_id)
+        if not app.langfuse_public_key or not app.langfuse_secret_key:
+            return ItemCreateResponse(
+                success=False,
+                message=f"Langfuse was not configured correctly application {interaction.app_id}.",
+            )
+
+        # create score
+        langfuse = Langfuse(
+            public_key=app.langfuse_public_key,
+            secret_key=app.langfuse_secret_key,
+        )
+        langfuse.score(
```

Comment:
It returns a StatefulClient. It seems that line 336 runs whenever langfuse.score is successful or failed.

Reply:
updated